internal/tsc_wrapped/compiler_host.ts

```diff
@@ -398,7 +398,7 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
    * looking under `bazelOpts.nodeModulesPrefix`
    */
   resolveTypeReferenceDirectives(names: string[], containingFile: string): ts.ResolvedTypeReferenceDirective[] {
```

Comment:
consider changing the type of `names` to `string[]|undefined` to better reflect the actual usage?

Reply:
sorry I was wrong, the undefined was lower. I'll do some more testing before sending the PR (wanted a PR number for the patch in rules_nodejs)